app/templates/static/layout.static.pug

```diff
@@ -73,9 +73,9 @@ html(lang='en')
               // IE10 warning
               var htmlElement = document.querySelector("html");
               if (htmlElement) {
-                  if ($.browser.msie && $.browser.versionNumber < 11) {
-                      if ((window.location.pathname || "").indexOf("web-dev-level") < 0) {
-                          alert("CodeCombat does not run in Internet Explorer 10 or older. Sorry!");
+                  if ($.browser.msie) {
```

Comment:
What is `$.browser.versionNumber` for edge?

Reply:
$.browser.versionNumber returns Edge's version number. 
As Andrew suggested, I can check for $.browser.edge defined or not.
For Edge both $.browser.msie and $.browser.edge are true
For IE, the latter is undefined